server/config/initializers/cors.rb

```diff
@@ -5,16 +5,16 @@
 
 # Read more: https://github.com/cyu/rack-cors
 
- Rails.application.config.middleware.insert_before 0, Rack::Cors do
-   allow do
-      origins 'https://alces-flight.com',
-        /^.*\.alces-flight.com$/,
-        /^.*\.alces-flight.lvh.me(:[0-9]+)?$/,
-        /^http:\/\/localhost(:[0-9]+)?$/
+Rails.application.config.middleware.insert_before 0, Rack::Cors do
+  allow do
+    origins "https://alces-flight.com",
+      /^.*\.concertim.com$/,
+      /^.*\.alces-flight.lvh.me(:[0-9]+)?$/,
```

Comment:
At a minimum could we get this `lvh.me` line duplicated for concertim.

Or perhaps duplicating all of them would be worth doing?  Or doing something similar to https://github.com/alces-flight/concertim-sso/blob/main/config/initializers/cors.rb ?